src/azul/health.py

```diff
@@ -0,0 +1,47 @@
+import boto3
+
+from botocore.exceptions import ClientError
+
+from azul import config
+
+
+def get_elasticsearch_health():
+    is_es_reachable = config.es_connection().ping()
+
+    result = {
+        'status': 'UP' if is_es_reachable else 'DOWN',
+        'domain': config.es_domain
+    }
+    if not is_es_reachable:
+        result['message'] = 'Unable to reach the host'
+    return result
+
+
+def get_queue_health():
+    notify_queue_name = "notifyQueue"
```

Comment:
I mentioned this before in the other PR: Generally, please don't extract expressions that are only used once. With a dict comprehension this can be made into a one liner.